doc/reportv2.Rmd

```diff
@@ -218,21 +222,14 @@ ggplotly(colourplz)
+The differences in predator-prey body size relationships suggests that different environment's have different capacities to produce resources, which thus determines how much energy is allowed to flow between predators and prey. For example, we found that piscivorous feeders in the seasonal pack ice zone have a lower intercept than piscivorous feeders in the shelf (fig. 4). Assuming that piscivores require similar energy needs, this supports the prediction that habitats with low productivity hold less energy, and thus produce food webs in which predators and prey tend to be smaller (Barnes *et al.*, 2010).
 
-Several limitations have been identified that may be important in the interpretation of this work. First, while this dataset had over 30,000 entries, much of the data represented nonindependent records. There were multiple records for the same predator, and further, multiple records for the same species that we took into account using a linear mixed effects model. In spite of this, nonindependent observations may have decreased the sample size, thus making it unrepresentative of food webs in marine habitats. For example, out of the whole dataset, 3581 samples are of the Albacore tuna. Statistically, these samples represent a single unit -- consequently, instead of having 3581 samples, there is only 1 sample with 3581 values. As a result, this dataset was limited in what it could actually communicate in terms of how predator-prey body size relationships vary across marine habitats and feeding types. Second, due to the manner in which environmental data was obtained for each dataset, we did not directly include it in our analysis. Instead, because environmental variables correlate with certain habitats and locations, we used specific habitat as a proxy for its effect on body size relationships. It was then not possible to infer exactly what environmetal factors were important in determining predator-prey body size relationships.
+However, because habitat can encompass a wide range of variables, it was not possible to parse out exactly what variables were important in predicting predator-prey size relationships. This was due to the manner in which environmental data was collected in this dataset. For example, the average temperature for each environment only included sea surface temperature, which was obtained from an extrinsic source (Barnes *et al.*, 2008). As a result, the temperature variable does not take into account changes in temperature with depth. Thus, for samples that were observed deep in the water column, sea surface temperature would not have been informative as a predictive variable. The same logic can be applied to other variables such as the average primary productivity of an environment, which was also measured at sea surface.
```

Comment:
Good discussion of the limitations of the dataset instead of trying to force drawing conclusions about it.